itest/Dockerfile.ubuntu_xenial

```diff
@@ -1,4 +1,4 @@
 FROM ubuntu:xenial
-ADD dist /work/dist
+ADD dist/ubuntu_xenial/ /work/dist/
 ADD deb_package_itest.sh /work/deb_package_itest.sh
-RUN /work/deb_package_itest.sh setup
\ No newline at end of file
+RUN /work/deb_package_itest.sh setup
```

Comment:
Couldn't this be make into a single file with the bionic version using the `HOSTRELEASE` argument?

Reply:
Yeah I've opened an issue for this already - I'll come back round to it once I've pushed this stuff out because it's taken me too long already!